sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -190,8 +194,8 @@ export interface CommonModelCreationOptions {
 export interface CopyAuthorization {
     accessToken: string;
     expirationDateTime: Date;
-    targetModelId: string;
-    targetModelLocation: string;
+    targetDocumentModelId: string;
+    targetDocumentModelLocation: string;
```

Comment:
This won't actually work at runtime. I think `targetModelId`/`Location` are enough in context.